### PR TITLE
docs: adding signal inputs and model to ADEV.

### PR DESCRIPTION
**Pull request by JeanMeche**
Those docs are a copy from AIO.

Direct link: https://ng-dev-previews-fw--pr-angular-angular-54753-adev-prev-kfdxxac0.web.app/guide/signals

### CODE REVIEW

File: adev/src/content/guide/signals/inputs.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# Signal inputs
+
+Signal inputs allow values to be bound from parent components.
+Those values are exposed using a `Signal` and can change during the lifecycle of your component.
+
+<div class="alert is-helpful">
+
+Signal inputs are currently in [developer preview](/guide/releases#developer-preview).
+
+</div>
```
Comment (twerske) [inline adev/src/content/guide/signals/inputs.md:10]: These need to be converted to the correct syntax

File: adev/src/content/guide/signals/inputs.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+<my-custom-comp disabled>
+```
+
+<div class="alert is-important">
+
+Do not use transforms if they change the meaning of the input, or if they are [impure](https://en.wikipedia.org/wiki/Pure_function#Impure_functions).
+
+Instead, use `computed` for transformations with different meaning, or an `effect` for impure code that should run whenever the input changes.
+
+</div>
```
Comment (twerske) [inline adev/src/content/guide/signals/inputs.md:144]: Same here

File: adev/src/content/guide/signals/model.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# Model inputs
+
+**Model inputs** are a special type of input that enable a component to propagate new values
+back to another component.
+
+<div class="alert is-helpful">
+
+Model inputs are currently in [developer preview](/guide/releases#developer-preview).
+
+</div>
```
Comment (twerske) [inline adev/src/content/guide/signals/model.md:10]: Same here

**Review by twerske — COMMENTED** [review]
Three super small comments about alert formatting in the new authoring format. Otherwise LGTM!

Thank you for this!

File: adev/src/content/guide/signals/model.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
+# Model inputs
+
+**Model inputs** are a special type of input that enable a component to propagate new values
+back to another component.
+
+HELPFUL: Model inputs are currently in [developer preview](/guide/releases#developer-preview).
+
+</div>
```
Comment (twerske) [inline adev/src/content/guide/signals/model.md:8]: nit: remove?

**Comment by twerske** [comment]
Feel free to move this to a non-draft PR and after the one fix this can be marked for merge!

**Review by twerske — APPROVED** [review]
LGTM with one nit!

Thank you SO MUCH @JeanMeche

File: adev/src/content/guide/signals/model.md (hunk 4 above)
Comment (JeanMeche) [reply adev/src/content/guide/signals/model.md:8]: Of course !!

**Review by twerske — APPROVED** [review]

**Comment by atscott** [comment]
This PR was merged into the repository by commit 09a2f3d5925c6ea78a540a498871d70735664b99.

**Pull request closed without merging**